Tests/FoundationEssentialsTests/ProcessInfoTests.swift

```diff
@@ -189,4 +189,16 @@ final class ProcessInfoTests : XCTestCase {
         processInfo.processName = originalProcessName
         XCTAssertEqual(processInfo.processName, originalProcessName)
     }
+
+#if os(Windows)
```

Comment:
Might as well run this test on all platforms - this behavior should be the same on macOS and Linux too